lib/ansible/modules/uri.py

```diff
@@ -121,6 +121,11 @@
           generated by supplying C(json) or C(form-urlencoded) for I(body_format).
     type: dict
     version_added: '2.1'
+  headers_scret:
```

Comment:
Add a changelog entry [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments) and integration test [here](https://github.com/ansible/ansible/tree/devel/test/integration/targets/uri).